HOW_TO_TEST.md

```diff
@@ -1,16 +1,26 @@
 # How to run tests
 
 ## Pre-requisites
+1. Ensure that your JAVA_HOME is set correctly
+    `export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_261.jdk/Contents/Home` 
+2. Download [SDK generation code](https://github.ibm.com/CloudEngineering/openapi-sdkgen/releases) 
+3. Clone [`GO SDK`] https://github.com/IBM/ibm-iae-go-sdk/
+4. Generate the sdk code with: 
+`openapi-sdkgen/openapi-sdkgen.sh  generate -g ibm-go -i ibmanalyticsengine-v3.json  -o /Users/git/ibm-iae-go-sdk  --genITs --genExamples --api-package github.com/IBM/ibm-iae-go-sdk`
```

Comment:
Lets add a note(hyper link) as to where this spec `ibmanalyticsengine-v3.json` can be downloaded from

Reply:
We also need to give some details on building the GO Module. 
The integration  tests have to run on the generated sdk and not the sdk from git. (Let have those instructions also as part of the readme)

Reply:
Added Cloud api docs.

Reply:
Added note to run the test on generated sdk